.github/workflows/pretter.yml

```diff
@@ -0,0 +1,37 @@
+name: Prettier Fix
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+      - 0.x
+  workflow_run:
+    workflows: ["Duster Fix"]
```

Comment:
Duster?

Reply:
If duster fixes code it may prevent other actions from committing their fix, due to limitations from GitHub.
So if it has run we want to run this action after duster has committed their change.
https://github.com/tighten/duster-action#:~:text=The%20resulting%20commit%20will%20not%20trigger%20another%20GitHub%20Actions%20Workflow%20run